css/style.css

```diff
@@ -6,25 +6,140 @@ a {
   text-decoration: none;
 }
 
+.social__link {
+  transition: .5s;
+}
+
 body {
   font-family: "Gotham Pro";
   font-weight: 400;
   font-size: 12px;
+  margin: 0;
+}
+
+.container {
+  max-width: 1030px;
+  margin: 0 auto;
+}
+
+.header {
+  min-height: 96px;
+  padding: 30px 0 8px 0;
+  display: flex;
+}
+
+.header__logo {
+  margin-right: 43px;
+}
+
+.header__logo-img {
+  width: 100%;
+  height: 100%;
+}
+
+.header__logo-mobile {
+  display: none;
+}
+
+.navbar {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar__item {
+  margin-right: 35px;
+}
+
+.navbar__left-side,
+.navbar__right-side {
+  display: flex;
+  align-items: center;
+}
+
+.navbar__right-side .navbar__item:nth-child(3)::after {
+  content: "1";
+  display: inline-block;
+  font-size: 9px;
+  text-align: center;
+  line-height: 16px;
+  color: #fff;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  background-color: #000;
+  margin-left: 4px;
+}
+
+.navbar__item:last-child {
+  margin-right: 0;
 }
 
-.link-more {
-  font-weight: 700;
-  text-transform: uppercase;
-} 
 .navbar__link {
   font-weight: 700;
   text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  transition: .4s;
+}
+
+.search-form {
+  width: 178px;
+  border-bottom: 1px solid #000;
+  margin-top: 6px;
+}
+
+.search-form__input {
+  width: 160px;
+  border: none;
+  outline: none;
+}
+
+.burger {
+  display: none;
 }
 
-.first-section__title {
+.market-section {
+  padding-top: 19px;
+}
+
+.banner {
+  min-height: 280px;
+  padding: 65px 0 0 58px;
+  background-image: url("../img/street.jpg");
+  /* border: 1px solid transparent; без этой границы верхний отступ заголовка 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)